model/metricset/_meta/fields.yml

```diff
@@ -818,6 +818,48 @@
           type: group
           description: Metrics and limits for the cgroup, collected by APM agents on Linux.
           fields:
+          - name: cpu
+            type: group
+            description: CPU-specific cgroup metrics and limits.
+            fields:
+            - name: id
+              type: long
+              description: ID for the current cgroup CPU.
+            - name: cfs
+              type: group
+              description: Completely Fair Scheduler (CFS) cgroup metrics.
+              fields:
+              # TODO: `shares` wasn't listed in the linked github issue, but it
+              # is still being reported by the agent. Was its omission an
+              # oversight?
```

Comment:
Fine to add when agents are sending it; which agent did you find that has this implemented already?

Reply:
Discussed in stand-up -- I found this as something being stored in https://github.com/elastic/gosigar/blob/master/cgroup/cpu.go, but not necessarily in any agent. This will be removed for now.